#inputs
float64 h_offset
---
#outputs
bool h_set
